readthedocs_theme/templates/includes/topnav.html

```diff
@@ -1,6 +1,7 @@
 {% macro menu_main() %}
   <a class="item{% if page and page.slug == 'features' %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/features/">Features</a>
   <a class="item{% if page and page.slug == 'pricing' %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/pricing/">Pricing</a>
+  <a class="item{% if page and 'blog' in page.slug %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/blog/">Blog</a>
```

Comment:
This is okay for now, but it also doesn't feel like a great top level menu item. I'd like to revisit re-re-naming the "Features" and turning that into a dropdown menu to help reduce bloat on the top menu.

Reply:
Yea -- would be nice to have `Resources` or similar as a top level with `Blog`, `Docs`, `Support` in there probably.